File: src/main/java/us/n8l/duplicatefinder/digests/SHA256.java
package us.n8l.duplicatefinder.digests;

import org.jetbrains.annotations.NotNull;

public class SHA256 extends BaseDigest {
    public SHA256(@NotNull byte[] bytes) {
        super(bytes);
    }

    @Override
    protected Algorithm getAlgorithm() {
        return Algorithm.SHA256;
    }
}

File: src/main/java/us/n8l/duplicatefinder/digests/SHA256.kt
package us.n8l.duplicatefinder.digests

import java.io.InputStream

class SHA256 : BaseDigest {
    constructor(bytes: ByteArray) : super(bytes)

    constructor(inStream: InputStream) : super(inStream)

    override fun getAlgorithm() = Algorithm.SHA256
}
